firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -359,6 +360,42 @@ int im21_door_id_set(String command) {
     return (int)strtol(buffer, NULL, 16);
 }
 
+int toggle_ble_antenna(String command) {
```

Comment:
I suggest changing the name to give some indication of what 1 and 0 mean. Since 1 (which normally means true) means the External Antenna and 0 (which normally means false) is the Internal Antenna, what do you think about calling this `toggle_external_ble_antenna`?

Reply:
maybe "use_external_ble_antenna"?